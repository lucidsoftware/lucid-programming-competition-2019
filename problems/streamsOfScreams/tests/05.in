17
ghost
boo!
ghost
boo!
ghost
ghost
boo!
ghost
ghost
ghost
boo!
ghost
ghost
ghost
ghost
ghost
boo!