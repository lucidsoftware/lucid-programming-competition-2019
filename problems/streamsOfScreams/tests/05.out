Ah!
Ahhh!!!
Ah!
Ahhh!!!
Ah!
Ah!
Ahhh!!!
Ah!
Ah!
Ah!
Ahhh!!!
Ah!
Ah!
Ah!
Ah!
Ah!
Ahhh!!!
27